From 033261799f3e3d2ab19b23ac9123cd18096fdda6 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Thu, 9 Nov 2023 17:18:24 +0100
Subject: [PATCH 09/29] ia64: Reserve map_shadow_stack() syscall number

map_shadow_stack was introducted after ia64 was removed from
the kernel hence it was not added to ia64 syscall table.

Fixes: 2fd0ebad27bc ("arch: Reserve map_shadow_stack() syscall number
for all architectures")
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index ee4311d08593..6d4fb0ebcb76 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -373,3 +373,4 @@
 450	common	set_mempolicy_home_node		sys_set_mempolicy_home_node
 451	common	cachestat			sys_cachestat
 452	common	fchmodat2			sys_fchmodat2
+453	common	map_shadow_stack		sys_map_shadow_stack
-- 
2.25.1

